SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/ChannelGroup.cs

```diff
@@ -9,19 +9,30 @@ namespace Microsoft.AppCenter.Channel
 {
     public sealed class ChannelGroup : IChannelGroup, IAppSecretHolder
     {
+        private static readonly TimeSpan WaitStorageTimeout = TimeSpan.FromSeconds(5);
```

Comment:
Shouldn't it be _waitStorageTimeout?

Reply:
Nope I guess, this is constant, not class field.